fuse/fuse.cpp

```diff
@@ -38,6 +38,8 @@
 // For unshare and bind mount
 #include <sched.h>
 #include <sys/mount.h>
+#include <sys/prctl.h>
+#include <sys/types.h>
```

Comment:
It looks like some of the build targets (like debian) want `#define _BSD_SOURCE` for using major/minor with sys/types.h.
I'm suprised about this, as on a ubuntu 16.04 it doesn't require it

Reply:
an musl on alpine wants something else too